external-crates/move/crates/move-compiler/src/naming/ast.rs

```diff
@@ -1583,6 +1588,7 @@ impl AstDebug for Type_ {
             }
             Type_::Var(tv) => w.write(format!("#{}", tv.0)),
             Type_::Anything => w.write("_"),
+            Type_::Void => w.write("_"),
```

Comment:
Same as above, I eventually see the syntax probably being `!`, though we can do whatever